addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/handler/LoxoneMiniserverHandler.java

```diff
@@ -0,0 +1,574 @@
+                    if ((OnOffType) command == OnOffType.ON) {
+                        if (control instanceof LxControlPushbutton) {
+                            ((LxControlPushbutton) control).pulse();
+                        } else {
+                            ((LxControlSwitch) control).on();
+                        }
+                    } else {
+                        ((LxControlSwitch) control).off();
+                    }
+                }
+                return;
+            }
+
+            if (control instanceof LxControlJalousie) {
+                LxControlJalousie jalousie = (LxControlJalousie) control;
+                if (command instanceof PercentType) {
+                    jalousie.moveToPosition(((PercentType) command).doubleValue() / 100);
+                } else if (command instanceof UpDownType) {
+                    if ((UpDownType) command == UpDownType.UP) {
+                        jalousie.fullUp();
+                    } else {
+                        jalousie.fullDown();
+                    }
+                } else if (command instanceof StopMoveType) {
+                    if ((StopMoveType) command == StopMoveType.STOP) {
+                        jalousie.stop();
+                    }
+                }
+                return;
+            }
+
+            if (control instanceof LxControlLightController) {
+                LxControlLightController controller = (LxControlLightController) control;
+                if (command instanceof OnOffType) {
+                    if ((OnOffType) command == OnOffType.ON) {
+                        controller.allOn();
+                    } else {
+                        controller.allOff();
+                    }
+                } else if (command instanceof UpDownType) {
+                    if ((UpDownType) command == UpDownType.UP) {
+                        controller.nextScene();
+                    } else {
+                        controller.previousScene();
+                    }
+                } else if (command instanceof DecimalType) {
+                    controller.setScene(((DecimalType) command).intValue());
+                }
+                return;
+            }
+
+            if (control instanceof LxControlRadio) {
+                LxControlRadio radio = (LxControlRadio) control;
+                if (command instanceof OnOffType) {
+                    if ((OnOffType) command == OnOffType.OFF) {
+                        radio.setOutput(0);
+                    }
+                } else if (command instanceof DecimalType) {
+                    radio.setOutput(((DecimalType) command).intValue());
+                }
+                return;
+            }
+
+            logger.debug("Incompatible operation on control {}", control.getUuid().toString());
+
+        } catch (IOException e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+        } catch (Exception e) {
```

Comment:
Here the strategy I assumed was that in case of an error in processing some control (due to code bugs), user should be still able to have the binding functional, just without the control that caused errors. Lack of this control can be submitted as a bug, or just ignored, if not important. I agree that this may cause bugs be not reported, but I think it is more valuable from the user point of view to have a functional binding without some control than to wait for a fix to a problem in a control he/she possibly does not care for.